php-8.0/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -e
+
+echo "Starting entrypoint script"
+CONF_DIR="/usr/local/etc/php/conf.d"
```

Comment:
I _think_ you could turn this into an `ENV` directive in the Dockerfile, and then remove the entrypoint script entirely. No strong feelings on whether that's worth doing, but deleting the entrypoint script is a great way to never have bugs in the entrypoint script.

Reply:
will do. This looks kinda pointless without newrelic ini file getting templated here.